packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -487,7 +496,10 @@ const helperBehavior = {
         }
       }
     }
+
     for await (const next of eventualLiquidations()) {
+      // This nested await is safe because "top-of-for-await"
```

Comment:
We should just add the top-level of a `for-await` as an allowed context for `awaits`

Reply:
Oh yes! Once we start fiddling with the static checks, the top of a for-await body should be jessie-safe.

The terminal-control-flow reason for safety is the most commonly used, should be straightforward to automate for the common cases, and covers some of the most useful cases, like many awaits in top level terminal try blocks. (comment from our slack repeated here for the record).